t/mojolicious/002_courses.t

```diff
@@ -124,8 +124,8 @@ $t->get_ok('/webwork3/api/courses/4/settings')->status_is(200)->content_type_is(
 $t->post_ok('/webwork3/api/courses' => json => $new_course)->status_is(403)->json_is('/has_permission' => 0);
 
 $t->put_ok('/webwork3/api/courses/4' => json => { course_name => 'XXX' })->status_is(403)
-	->json_is('/has_permission' => false);
+	->json_is('/has_permission' => 0);
```

Comment:
This seems like this should be a boolean and can then this with a switch on line 95 in `WeBWorK3::Controller::Permission` to `has_permission => false`.

Reply:
This would then affect almost all of the other mojolicious tests as well though.

Reply:
I am not going to change this in this pull request.  This was changed to 0 because currently the method returns a numeric (not boolean) value.  That is a TODO for later.